src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
+      const isEnabled = !matches(n, DISABLED_PSEUDOCLASS_);
```

Comment:
@robinpokorny Hmm, I think I'm missing the point. E.g. in my quick tests:

```
// Start with:
// Markup: <button disabled></button>
button.disabled === true

// Update property:
button.disabled = false
// Now:
// Markup: <button></button> ("disabled" attribute removed)
button.disabled === false

// Add attribute:
button.setAttribute('disabled', '');
// Now:
// Markup: <button disabled></button>
button.disabled === true
```

Isn't this exactly what we want?